<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lbl_Test">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>This is Label - Display Text</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_changeText">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>160</y>
     <width>116</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>ChangeText</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_printText">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>98</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>PrintText</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
